### ISSUE TITLE
NTR: Calcified mediastinal lymph nodes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10619]
**Preferred term label:**
Calcified mediastinal lymph nodes

**Synonyms**
Lymph node calcification

**Definition (free text, please give PubMed ID)**
PMID: 29791561
Lymph node calcifications most often result from prior granulomatous infections, especially tuberculosis and histoplasmosis. Other, less common, causes are sarcoidosis, silicosis, amyloidosis, and calcifications secondary to the treatment of lymphomas (radiation therapy or chemotherapy).

**Parent term (use hpo.jax.org/app)**
Abnormal lymph node morphology HP:0002733

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
Adding new term: 
- ID: HP:6001018
- label: Mediastinal lymph node calcification
- parents: [Abnormal lymph node morphology (HP:0002733)](https://hpo.jax.org/app/browse/term/HP:0002733)
- pmid: PMID:29791561
- def: Abnormal deposition of calcium in mediastinal lymph nodes.
- comment: Lymph node calcifications most often result from prior granulomatous infections, especially tuberculosis and histoplasmosis. Other, less common, causes are sarcoidosis, silicosis, amyloidosis, and calcifications secondary to the treatment of lymphomas (radiation therapy or chemotherapy).
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10619
- synonyms: Calcified mediastinal lymph nodes(NONE)
